src/Uno.Toolkit.RuntimeTests/Tests/CardContentControlTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.Toolkit.RuntimeTests.Extensions;
+using Uno.Toolkit.RuntimeTests.Helpers;
+using Uno.Toolkit.UI;
+using Uno.UI.RuntimeTests;
+using Uno.UI.Extensions;
+using Windows.Foundation;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation.Metadata;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+using Microsoft.UI;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+#else
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml;
+using Windows.UI;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+#endif
+
+namespace Uno.Toolkit.RuntimeTests.Tests
+{
+	[TestClass]
+	[RunsOnUIThread]
+	internal class CardContentControlTests
+	{
+		[TestMethod]
+		[RequiresFullWindow]
+		public async Task Should_Hug_Content()
+		{
+			var rootGrid = XamlHelper.LoadXaml<Grid>(@$"
+				<Grid xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml"" Width=""500"" Height=""500"">
+					<utu:CardContentControl Padding=""0"" x:Name=""MyCard"" Style=""{{StaticResource FilledCardContentControlStyle}}"">
+						<Grid Background=""Red"" Height=""200"" Width=""200"" />
+					</utu:CardContentControl>
+				</Grid>
+			");
```

Comment:
you can use [`"""` raw string literal](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/raw-string-literal) to avoid having to escape `"`
it support $tring-intepolation too, do note that if you need to use escape `{}`, you will have to do this:
```cs
$$"""
<!-- note: amount of interpolate brace is equal to amount of $ you used -->
<Border prop1="{{value}}" prop2="{Binding}" />
"""
```

Reply:
XamlHelper.LoadXaml has several features:
1. you can also use single quote `'` for xaml attribute in here, eg: `"<Border Width='312' />"`
2. you can omit common xmlns declaration, eg: , eg: `"<utu:Chip x:Name='asd' />"`